test/web-platform-tests/to-upstream/html/browsers/windows/nested-browsing-contexts/dont-upstream-frameElement.html

```diff
@@ -0,0 +1,32 @@
+<!-- Based on https://github.com/web-platform-tests/wpt/blob/203c34855f6871d6e55eaf7b55b50dad563f781f/html/browsers/windows/nested-browsing-contexts/frameElement.sub.html -->
```

Comment:
Previous tests have had `dont-upstream` appended (rather than prepended) to their filename. Could be helpful to continue with the same pattern.